lib/Marlin/Marlin/src/module/temperature.cpp

```diff
@@ -3842,7 +3844,11 @@ void Temperature::isr() {
     #endif
     #if HAS_HEATED_CHAMBER
       SERIAL_ECHOPAIR(" C@:", getHeaterPower(H_CHAMBER));
+    #elif HAS_CHAMBER_API()
+      auto current_chamber_temperature = buddy::chamber().current_temperature();
+      if (buddy::chamber().capabilities().temperature_reporting && current_chamber_temperature.has_value()) SERIAL_ECHOPAIR(" C@:", current_chamber_temperature.value());
```

Comment:
No need to check for capabilities here. If the chamber does not have the capability, the temperature will not have a value.